packages/react-testing/src/index.ts

```diff
@@ -4,3 +4,5 @@ export type {Node} from './types';
 export * from './mount';
 export * from './destroy';
 export * from './toReactString';
+export {createMock} from 'ts-jest-mock';
```

Comment:
I don't like us adding jest-specific stuff to this entrypoint, currently everything jest related is cordoned off in the matchers one.